README.md

```diff
@@ -23,7 +23,7 @@ To get started with the quickstart application follow these steps. Steps 1-6 wil
 12. [Make client to PSTN call](#bullet12)
 
 ### <a name="bullet1"></a>1. Install the TwilioVoice framework using Cocoapods
-Under the quickstart path, run `pod install` and let the Cocoapods library create the workspace for you. Also please make sure to use **Cocoapods v1.0 and later**.
+Under the quickstart path, run `pod update` and `pod install` and let the Cocoapods library create the workspace for you. Also please make sure to use **Cocoapods v1.0 and later**.
```

Comment:
This is OK however [this](https://github.com/twilio/video-quickstart-objc#cocoapods) is how we have this in video